src/lib/supported-node-js-version-check.js

```diff
@@ -0,0 +1,46 @@
+// Ref. https://nodejs.org/en/about/releases
+const nodeVersionToEndOfLifeDateMap = {
+	'12': new Date('2022-04-30'),
+	'14': new Date('2023-04-30'),
+	'16': new Date('2023-09-11'),
+	'18': new Date('2025-04-30'),
+	'20': new Date('2026-04-30')
+};
+
+/**
+ * @param {string} appName
+ * @returns {import("../../typings/metrics").Healthcheck}
+ */
+module.exports = (appName) => {
+	const lastUpdated = new Date();
+
+	const currentDate = new Date();
```

Comment:
I tried to use `const lastUpdated = currentDate = new Date();` here but it failed the linting rules (`no-undef`).